.eslintrc.yaml

```diff
@@ -34,6 +34,7 @@ overrides:
   - files: ['*.spec.js', '**/docs/*.js']
     rules:
       'react/display-name': 0
+      'react/prop-types': 0
```

Comment:
why?

Reply:
In test files and usage example files I find it's not really necessary to have, it's just more boilerplate.

Reply:
I agree. It's about the test cases less about always being complete. Plus testing for incomplete `prop-types` has a propose.